pytest_splunk_addon/plugin.py

```diff
@@ -106,15 +106,15 @@ def init_pytest_splunk_addon_logger():
     Configure file based logger for the plugin
     """
     fh = logging.FileHandler(LOG_FILE)
-    fh.setLevel(logging.DEBUG)
```

Comment:
if we can't parameterize it, let's keep it DEBUG for now.
@jayjoshi-crest Views?

Reply:
Alright, change has been reverted